integrity-backend/input-starling-capture-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-content.json

```diff
@@ -0,0 +1,50 @@
+    },
+    "private":
+    {
+      "meta": "eyJwcm9vZiI6eyJoYXNoIjoiN2IyNzkzMzQzMWY1ZWIyNThiOGU0Y2VhNzE4MjE5MDMxZDdhNTgyMTA4NjQ0ZTdmYzliZWVlNmE5MDU1ZTQ3YiIsIm1pbWVUeXBlIjoiaW1hZ2UvanBlZyIsInRpbWVzdGFtcCI6MTY1MDU2NTY2NTM5OX0sImluZm9ybWF0aW9uIjpbeyJwcm92aWRlciI6IkluZm9TbmFwc2hvdCIsIm5hbWUiOiJBY2NlbGVyb21ldGVyIiwidmFsdWUiOiJbLTAuMDg1NDQ5MjIsIDAuMDI3MTMwMTI3LCA5Ljc0OTIyMV0ifSx7InByb3ZpZGVyIjoiSW5mb1NuYXBzaG90IiwibmFtZSI6IkJvYXJkIiwidmFsdWUiOiJRQ19SZWZlcmVuY2VfUGhvbmUifSx7InByb3ZpZGVyIjoiSW5mb1NuYXBzaG90IiwibmFtZSI6IkJyYW5kIiwidmFsdWUiOiJIdGMifSx7InByb3ZpZGVyIjoiSW5mb1NuYXBzaG90IiwibmFtZSI6IkRldmljZSBCdWlsZCBGaW5nZXJwcmludCIsInZhbHVlIjoiSHRjL2JyZTJleGR1Z2xfMDI0MDEvaHRjX2JyZTJleGR1Z2w6OC4xLjAvT1BNMS4xNzEwMTkuMDExLzIwMDkwNDp1c2VyL3JlbGVhc2Uta2V5cyJ9LHsicHJvdmlkZXIiOiJJbmZvU25hcHNob3QiLCJuYW1lIjoiRGV2aWNlIEJ1aWxkIElEIiwidmFsdWUiOiIxLjE2LjI0MDEuMiJ9LHsicHJvdmlkZXIiOiJJbmZvU25hcHNob3QiLCJuYW1lIjoiRGV2aWNlIEJ1aWxkIFRhZ3MiLCJ2YWx1ZSI6InJlbGVhc2Uta2V5cyJ9LHsicHJvdmlkZXIiOiJJbmZvU25hcHNob3QiLCJuYW1lIjoiRGV2aWNlIEJ1aWxkIFRpbWUiLCJ2YWx1ZSI6IjA0LzA5LzIwMjAgMDk6MzIifSx7InByb3ZpZGVyIjoiSW5mb1NuYXBzaG90IiwibmFtZSI6IkRldmljZSBCdWlsZCBUeXBlIiwidmFsdWUiOiJ1c2VyIn0seyJwcm92aWRlciI6IkluZm9TbmFwc2hvdCIsIm5hbWUiOiJEZXZpY2UgTmFtZSIsInZhbHVlIjoiaHRjX2JyZTJleGR1Z2wifSx7InByb3ZpZGVyIjoiSW5mb1NuYXBzaG90IiwibmFtZSI6IkVuZCBQcm9kdWN0IE5hbWUiLCJ2YWx1ZSI6IkVYT0RVUyAxcyJ9LHsicHJvdmlkZXIiOiJJbmZvU25hcHNob3QiLCJuYW1lIjoiR2FtZSBSb3RhdGlvbiBWZWN0b3IiLCJ2YWx1ZSI6IlVOU1VQUE9SVEVEIn0seyJwcm92aWRlciI6IkluZm9TbmFwc2hvdCIsIm5hbWUiOiJHZW9tYWduZXRpYyBSb3RhdGlvbiBWZWN0b3IiLCJ2YWx1ZSI6Ils0LjQ4NDcyMkUtNCwgMC4wMDUzODcyOTUsIDAuMDM5ODA4NjI3LCAwLjk5OTIwMjYsIDAuMF0ifSx7InByb3ZpZGVyIjoiSW5mb1NuYXBzaG90IiwibmFtZSI6IkdyYXZpdHkiLCJ2YWx1ZSI6IlstMC4xMDUyMjUxOCwgMC4wMTI5OTUwNjMsIDkuODA2MDYzXSJ9LHsicHJvdmlkZXIiOiJJbmZvU25hcHNob3QiLCJuYW1lIjoiR3lyb3Njb3BlIiwidmFsdWUiOiJVTlNVUFBPUlRFRCJ9LHsicHJvdmlkZXIiOiJJbmZvU25hcHNob3QiLCJuYW1lIjoiSGFyZHdhcmUiLCJ2YWx1ZSI6InFjb20ifSx7InByb3ZpZGVyIjoiSW5mb1NuYXBzaG90IiwibmFtZSI6Ikhhc2ggb2YgQW5kcm9pZCBJRCIsInZhbHVlIjoiZjI5MDlhZWU2ZGM5OGFmNjc5MTExNWI1NzUxYzIwNDYwOGIwNTg4M2RjNmZjNGJlYmUwOWE3MjE2NDU2MWVkZiJ9LHsicHJvdmlkZXIiOiJJbmZvU25hcHNob3QiLCJuYW1lIjoiTGlnaHQiLCJ2YWx1ZSI6IlsxMy4wXSJ9LHsicHJvdmlkZXIiOiJJbmZvU25hcHNob3QiLCJuYW1lIjoiTGluZWFyIEFjY2VsZXJvbWV0ZXIiLCJ2YWx1ZSI6IlstMC4wMDUxNDEyNjEsIDAuMDE0MTc4MjksIC0wLjA0MzExMzM0N10ifSx7InByb3ZpZGVyIjoiSW5mb1NuYXBzaG90IiwibmFtZSI6Ik1hZ25ldGljIEZpZWxkIiwidmFsdWUiOiJbMS40NjQ4NDM4LCA0Ny4yNjU2MjUsIC05My4wNjUzNF0ifSx7InByb3ZpZGVyIjoiSW5mb1NuYXBzaG90IiwibmFtZSI6Ik1hbnVmYWN0dXJlciIsInZhbHVlIjoiSFRDIn0seyJwcm92aWRlciI6IkluZm9TbmFwc2hvdCIsIm5hbWUiOiJPdmVyYWxsIFByb2R1Y3QgTmFtZSIsInZhbHVlIjoiYnJlMmV4ZHVnbF8wMjQwMSJ9LHsicHJvdmlkZXIiOiJJbmZvU25hcHNob3QiLCJuYW1lIjoiUm90YXRpb24gVmVjdG9yIiwidmFsdWUiOiJbNC40ODQ3MjMzRS00LCAwLjAwNTM4NzI5NSwgMC4wMzk4MDg1OTcsIDAuOTk5MjAyNTUsIDAuMF0ifSx7InByb3ZpZGVyIjoiSW5mb1NuYXBzaG90IiwibmFtZSI6IlRpbWVzdGFtcCIsInZhbHVlIjoiMjAyMi0wNC0yMVQxODoyNzo1Ni4yMDRaIn1dfQ==",
```

Comment:
@makeworld-the-better-one this is the `meta` field, unmodified. This should fix https://github.com/starlinglab/integrity-backend/issues/91

Reply:
I was fixing that issue [like this](https://github.com/starlinglab/integrity-backend/pull/92/files#diff-b582c00a7706964259c9860d789f2c27850be778f617760f11967813fb540a4c). I'm not sure how this meshes with your solution.